macros/tests/json.rs

```diff
@@ -1,6 +1,5 @@
 extern crate gll;
 extern crate gll_macros;
```

Comment:
Are these really needed?

Reply:
I couldn't get rid of it :( Perhaps try to remove them after this PR since you're more familiar?

Reply:
I found the reason, it's elsewhere.